src/OpenTelemetry.Exporter.Stackdriver/Implementation/ActivityExtensions.cs

```diff
@@ -14,6 +14,8 @@
 // limitations under the License.
 // </copyright>
 
+#nullable enable
```

Comment:
This switch can be removed later.
IMO it is better to have some annotation instead of removing it now and reintroducing in next PRs.